lib/easypost/http_client.rb

```diff
@@ -27,11 +27,16 @@ def request(
     request.body = JSON.dump(EasyPost::InternalUtilities.objects_to_ids(body)) if body
 
     # Execute the request, return the response.
-    Net::HTTP.start(
-      uri.host,
-      uri.port, use_ssl: true, read_timeout: @config[:read_timeout], open_timeout: @config[:open_timeout],
-    ) do |http|
-      http.request(request)
+
```

Comment:
nit: please attach the comment to its associated logic

Reply:
Resolved